# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/wgpu/player/tests/data/bind-group.ron
# Commit: ef5dc3e04e5f
# Full Hash: ef5dc3e04e5f271eea0636ab3a495e95cc912f1d
# Author: Dzmitry Malyshau <dmalyshau@mozilla.com>
# Date: 2021-09-04 09:40:24
# Regressor Bug: 1726626
# File Overlap Count: 3
# Description:
#   Bug 1726626 - Move gfx/wgpu into a 3rd party dependency r=jgilbert,bholley
#   
#   This update makes wgpu a vendored dependency instead of having it in gfx/wgpu.
#   
#   ## Notes
# ==============================================================================

diff --git a/gfx/wgpu/player/tests/data/bind-group.ron b/gfx/wgpu/player/tests/data/bind-group.ron
deleted file mode 100644
--- a/gfx/wgpu/player/tests/data/bind-group.ron
+++ /dev/null
@@ -1,81 +0,0 @@
-(
-    features: (bits: 0x0),
-    expectations: [], //not crash!
-    actions: [
-        CreatePipelineLayout(Id(0, 1, Empty), (
-            label: Some("empty"),
-            bind_group_layouts: [],
-            push_constant_ranges: [],
-        )),
-        CreateShaderModule(
-            id: Id(0, 1, Empty),
-            desc: (
-                label: None,
-                flags: (bits: 3),
-            ),
-            data: "empty.wgsl",
-        ),
-        CreateComputePipeline(
-            id: Id(0, 1, Empty),
-            desc: (
-                label: None,
-                layout: Some(Id(0, 1, Empty)),
-                stage: (
-                    module: Id(0, 1, Empty),
-                    entry_point: "main",
-                ),
-            ),
-        ),
-        CreateBuffer(Id(0, 1, Empty), (
-            label: None,
-            size: 16,
-            usage: (
-                bits: 64,
-            ),
-            mapped_at_creation: false,
-        )),
-        CreateBindGroupLayout(Id(0, 1, Empty), (
-            label: None,
-            entries: [
-                (
-                    binding: 0,
-                    visibility: (bits: 0x3),
-                    ty: Buffer(
-                        ty: Uniform,
-                    ),
-                ),
-            ],
-        )),
-        CreateBindGroup(Id(0, 1, Empty), (
-            label: None,
-            layout: Id(0, 1, Empty),
-            entries: [
-                (
-                    binding: 0,
-                    resource: Buffer((
-                        buffer_id: Id(0, 1, Empty),
-                        offset: 0,
-                        size: None,
-                    )),
-                )
-            ],
-        )),
-        Submit(1, [
-            RunComputePass(
-                base: (
-                    commands: [
-                        SetPipeline(Id(0, 1, Empty)),
-                        SetBindGroup(
-                            index: 0,
-                            num_dynamic_offsets: 0,
-                            bind_group_id: Id(0, 1, Empty),
-                        ),
-                    ],
-                    dynamic_offsets: [],
-                    string_data: [],
-                    push_constant_data: [],
-                ),
-            ),
-        ]),
-    ],
-)
\ No newline at end of file